docs/architecture/fse-templates.md

```diff
@@ -30,6 +30,13 @@ The synchronization is important for two different flows:
  - When rendering a template (sometimes referred to as "resolving a template"): this is the algorithm that WordPress follows to traverse the template hierarchy and find the right template to render for the current page being loaded.
  - When exporting a block-based theme, we need to export all its templates back as files. The synchronizaion is required in order to simplify the operation and only export the CPT templates.
 
+Synchronization is triggered in the following situations:
```

Comment:
should we remove the above list then?

Reply:
Made a couple of changes so reverted my doc changes.